commit ec11389accb2b59431dd851b05b7a3095d609fb2 (refs/changes/69/4569/2)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T20:17:37+00:00 (1 year, 2 months ago)
    
    TRITON-592 when provisioning vmapi should check for network pools before networks (bump version)
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index c17bdd7..77c4a3a 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.5.3",
+  "version": "9.6.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
